package-lock.json

```diff
@@ -3126,7 +3135,6 @@
       "version": "2.1.4",
       "resolved": "https://registry.npmjs.org/@nodelib/fs.scandir/-/fs.scandir-2.1.4.tgz",
       "integrity": "sha512-33g3pMJk3bg5nXbL/+CY6I2eJDzZAni49PfJnL5fghPTggPvBd/pFNSgJsdAgWptuFu7qq/ERvOYFlhvsLTCKA==",
-      "dev": true,
```

Comment:
Does anyone know why these `"dev": true,` lines were removed?

Reply:
I'm guessing `plop` needs them, and it's currently an external dependency -- if we move it to a dev dependency these changes should be gone